### PR TITLE
Updating image version to 311.153.20191113 and create 10.2

### PR DESCRIPTION
**Pull request by kwoodson**
```release-note
Create v10.2 and new image
```

### CODE REVIEW

**Comment by jim-minter** [comment]
@kwoodson looking good but check unit test failures

**Comment by kwoodson** [comment]
@jim-minter Thanks.  I wasn't sure if I should modify this since the 10.1 key did not exist in the hash.

Will update.

**Comment by kwoodson** [comment]
@jim-minter @asalkeld PTAL

File: pluginconfig/pluginconfig-311.yaml [hunk 1]
```diff
@@ -220,6 +220,47 @@ versions:
       sync: osarpint.azurecr.io/openshift-on-azure/azure:v10.1
       tlsProxy: osarpint.azurecr.io/openshift-on-azure/azure:v10.1
       logAnalyticsAgent: mcr.microsoft.com/azuremonitor/containerinsights/ciprod:ciprod07092019
+  v10.2:
```
Comment (jim-minter) [inline pluginconfig/pluginconfig-311.yaml]: @kwoodson you need to set `pluginVersion: v10.2` at the top of this file

Comment (jim-minter) [reply pluginconfig/pluginconfig-311.yaml]: @kwoodson damn, it merged, please can you do a follow-up PR to do this?

**Pull request merged**